### PR TITLE
Awesome books website with Es-6 and modules

### PR DESCRIPTION
**Pull request by tonnytech**
Refactored the awesome books website with Es-6 syntax

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,48 +155,42 @@ Feel free to check the [issues page](../../issues/).
...
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (TadesseDev) [inline README.md]: - [ ] This part should not be part of your readme, this is just a direction on how to use a License file. Please update your readme accordingly.

File: README.md [hunk 2]
```diff
@@ -1,40 +1,13 @@
...
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
 
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+
+  <img src="./assets/img/murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (TadesseDev) [inline README.md]: - [ ] Please consider adding your logo to the readme file. If not feel free to remove it.

**Review by TadesseDev — COMMENTED** [review]
### Hi @teitonny 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

 - [ ] You used the recommended README template for your project but at the moment you did not fill it with contents related to this project or your information. Please remove any content that does not explain the current project or author.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @teitonny, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**